pkg/controller/controller.go

```diff
@@ -64,11 +63,26 @@ type Controller struct {
 
 func (c *Controller) CreateVolume(ctx *c.Context, in *model.VolumeSpec, errchanVolume chan error) {
 	var err error
-	var profile *model.ProfileSpec
+	var prf *model.ProfileSpec
```

Comment:
I'd prefer to spell it out as "profile", unless if there's any conflict.

Reply:
The reason I modified here is that we usually use `profile` to represent the instance itself, but for the point to the instance, we'd better to use other words.